examples/storm-kafka-client-examples/pom.xml

```diff
@@ -73,12 +73,6 @@
             <version>${project.version}</version>
             <scope>${provided.scope}</scope>
         </dependency>
-        <dependency>
```

Comment:
Do you run example topology after this change? If then I'm OK. Just to verify the change doesn't hurt anything.